src/Utilities/Resources/xlf/Strings.de.xlf

```diff
@@ -185,7 +185,7 @@
       </trans-unit>
       <trans-unit id="Tracking_SourceWillBeCompiledDependencyWasModifiedAt">
         <source>{0} will be compiled as {1} was modified at {2}.</source>
-        <target state="translated">{0} wird als {1} kompiliert, wurde geändert bei {2}.</target>
```

Comment:
👍 for changing the English text
If not, xlf has a note field for issues like this..